add-on/chrome/modules/LoopRooms.jsm

```diff
@@ -730,8 +733,9 @@ var LoopRoomsInternal = {
     }
   },
 
+  // XXX akita Bug 1268826 tidy up chat window code
   open: function(roomToken) {
-    let windowData = {
+    /* let windowData = {
```

Comment:
Just a thought: This change is going to break some of the xpcshell-tests. I'm ok with that, but maybe add an XXX to test_looprooms.js to fix after the separate window has landed?